packages/@aws-cdk/aws-glue-alpha/lib/jobs/pyspark-etl-job.ts

```diff
@@ -7,23 +7,10 @@ import { JobType, GlueVersion, JobLanguage, PythonVersion, WorkerType } from '..
 import { SparkUIProps, SparkUILoggingLocation, validateSparkUiPrefix, cleanSparkUiPrefixForGrant } from './spark-ui-utils';
 import { Code } from '../code';
 
-/**
- *  PySpark ETL Jobs class
- *  ETL jobs support pySpark and Scala languages, for which there are separate
- *  but similar constructors. ETL jobs default to the G2 worker type, but you
- *  can override this default with other supported worker type values
- *  (G1, G2, G4 and G8). ETL jobs defaults to Glue version 4.0, which you can
- *  override to 3.0. The following ETL features are enabled by default:
- *  —enable-metrics, —enable-spark-ui, —enable-continuous-cloudwatch-log.
- *  You can find more details about version, worker type and other features
- *  in Glue's public documentation.
- */
```

Comment:
1) these are docstrings, which are meant to be attached to a particular code construct. 
2) there's no reason to document source code like this. much of this can/should be in the README, or a docstring on the particular construct we vend to customers. that way they can see our documentation in their IDE, or in our published docs.